src/_premake_init.lua

```diff
@@ -204,6 +204,18 @@
 		tokens = true,
 	}
 
+	api.register {
+		name = "copytooutputdirectory",
```

Comment:
Could this be an extension to the `buildaction` instead? Some of the other new APIs also seem like they could be extensions of `buildaction`. Additionally, there isn't really any way to override the usage of these APIs, they appear to only be used in a single, ever growing function.

Reply:
Is this different than `copylocal`?